docs/components/llms.mdx

```diff
@@ -330,6 +330,7 @@ Setup Ollama using https://github.com/jmorganca/ollama
 
 ```python main.py
 import os
+os.environ["OLLAMA_HOST"] = "http://127.0.0.1:11434"
```

Comment:
@golemus What do you need to set host here?

Reply:
to the IP of your local ollama installation (localhost if same PC)
https://github.com/ollama/ollama/blob/main/docs/faq.md

Reply:
Alright, got it.